Makefile

```diff
@@ -76,7 +76,11 @@ check: test lint-golint lint-codespell
 
 # Run tests
 test: generate fmt vet manifests verify-image
-	go test -mod=readonly ./... -coverprofile cover.out
+	go test \
+	  -mod=readonly \
+	  -covermode=atomic \
+	  -coverprofile cover.out \
```

Comment:
@sunjayBhatia should this match `prbuild.yaml` and be `coverage.out`?

Reply:
good point, missed that copying over from the other repo